### PR TITLE
Update choose-repository.html

### PR DESCRIPTION
**Pull request by asad1983**
html changes

### CODE REVIEW

**Comment by asad1983** [comment]
I have changed the style for all section

**Comment by MicahZoltu** [comment]
Which issue does this fix?

**Comment by asad1983** [comment]
These changes is related to https://github.com/Zoltu/RepoCreator-Client/issues/1

File: dist/source/repo-creator/choose-repository.css [hunk 1]
```diff
@@ -145,3 +148,24 @@
...
+}
+
+}
+@media all and (max-width: 479px) and (min-width: 409px){
+.choose-repository .data-link {
+font-size: 12px !important;
+
+}
+
+}
```
Comment (MicahZoltu) [inline dist/source/repo-creator/choose-repository.css]: Can we remove these `font-size` changes?  I did a brief check and they don't appear to be necessary for the fix to work and I would rather not have multiple font sizes as the screen resizes.  The `text-overflow`, `padding-right` and `overflow` changes appear to have resolved the issue.

**Comment by MicahZoltu** [comment]
Looks good overall.  Just remove the font change sections and I think we are good to go.

I did notice that the first row of each list appears to have different padding/margin than all of the other rows.  I'll open another issue for that though since I don't think it is related to these changes.

**Pull request merged**